.circleci/config.yml

```diff
@@ -430,6 +431,60 @@ jobs:
       - store_artifacts:
           path: /home/circleci/project/test-reports/report.htm
 
+  integration_test_ios_fennec:
+    macos:
+      xcode: 15.3.0
+    steps:
+      - checkout
+      - macos/preboot-simulator:
+          version: "17.4"
+          platform: "iOS"
+          device: "iPhone 15"
+      - attach_workspace:
+          at: /tmp/experimenter
+      - run:
+          name: Clone Repo
+          command: git clone https://github.com/mozilla-mobile/firefox-ios.git
+      - run:
+          name: Install nimbus-cli
+          command: curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/mozilla/application-services/main/install-nimbus-cli.sh | bash
+      - run:
+          name: Setup and build Firefox app
+          command: |
+            cd firefox-ios
+            sh ./bootstrap.sh
+            cd firefox-ios
+            pip3 install virtualenv poetry
+            xcodebuild build-for-testing -project Client.xcodeproj -scheme Fennec -configuration Fennec -sdk iphonesimulator -quiet -destination 'platform=iOS Simulator,name=iPhone 15,OS=17.4'
+      - run:
+          name: Run Tests
+          command: |
+            cd firefox-ios/firefox-ios/firefox-ios-tests/Tests/ExperimentIntegrationTests
+            export SIMULATOR_UDID=$(python get_specific_device_udid.py)
+            cp -f /Users/distiller/project/experimenter/tests/integration/nimbus/ios/ios_pytest_config.py ./
+            cp -f /Users/distiller/project/experimenter/tests/integration/nimbus/ios/test_ios_integration.py ./
+            mv ios_pytest_config.py conftest.py
```

Comment:
* Can we put everything into one file?
* You can probably replace `/Users/distiller/` with `~`
* I don't know if `ln -s` is easier than `cp`?
* You can `cp src tgt` instead of `cp;mv`